backend/src/main/java/com/google/blackswan/servlets/CronServlet.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.blackswan.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.logging.Logger;
+
+/**
+* Servlet to test if cron job runs.
+*/
+@WebServlet("/blackswan/test")
+public class CronServlet extends HttpServlet {
+  private static final Logger log = Logger.getLogger(CronServlet.class.getName());
+  
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    log.info("Cron job ran.");
+    response.setStatus(200); 
```

Comment:
prefer no magic numbers

For HttpServletResponse, I think you can use their pvodied status response codes (https://docs.oracle.com/javaee/6/api/javax/servlet/http/HttpServletResponse.html) 

constants and their corresponding names: https://docs.oracle.com/javaee/6/api/constant-values.html#javax.servlet.http.HttpServletResponse.SC_OK

Reply:
Sounds good, my bad, I should've used that in the start.